_includes/specials/taproot/00-bech32m.md

```diff
@@ -0,0 +1,42 @@
+## Preparing for taproot #1: bech32 sending support
+{% assign trb = "709,632" %}{% comment %}<!-- stop me from mistyping the number -->{% endcomment %}
+
+*The first segment in a weekly series about how developers and service providers
+can prepare for the upcoming activation of taproot at block height
+{{trb}}.*
+
+Starting at block {{trb}}, expected in November, Bitcoin users will be
+able to safely receive payments to taproot  addresses.  Given the
+user enthusiasm for taproot and the five months that wallet
+developers have to implement support for it, Optech expects there to be
+several popular wallets that will allow their users to generate taproot
+addresses at the earliest possible moment.
+
+That means any other wallet or service that sends bitcoins to
+user-provided addresses needs to be able to Pay To TapRoot (P2TR) by
+block {{trb}} or risk confusing and disappointing its users.  Taproot
+addresses use [bech32m][topic bech32] as specified in [BIP350][], which
+is slightly different than [BIP173][]'s bech32 algorithm used for segwit
+v0 P2WPKH and P2WSH addresses.  The slight difference is that the
+checksum function uses a different constant, with bech32m using
+`0x2bc830a3` instead of bech32's `0x01`.
+
+Changing that single constant in your code will provide you with the
```

Comment:
Narrative shift from referring to wallets in the third person in the previous paragraphs to the second person ("you")...